polling_stations/apps/core/opening_times.py

```diff
@@ -1,9 +1,8 @@
 import datetime
+from collections import OrderedDict
 from datetime import date, time
 from typing import Dict, List
 
-from typing_extensions import OrderedDict
-
```

Comment:
@symroe `typing_extensions` weren't available in 3.10, I could've imported `OrderedDict` from the `typing` lib, but IIUC we're instantiating an new instance of the class, rather than asserting the type in line 11 (`self.dates: Dict[date, List] = OrderedDict()`) and so we want the class rather than the type in the import? FWIW it works either way, which makes me think I don't really understand what's going on.